src/request/components/request-detail-panel-webservices.jsx

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+var messageProcessor = require('../util/request-message-processor');
+
+var _ = require('lodash');
+var React = require('react');
+var PanelGeneric = require('./request-detail-panel-generic');
+var classNames = require('classnames');
+
+/**
+ * Return the messages to be used by the view.
+ */
+var getMessages = (function() {
+    var getList = messageProcessor.getTypeMessageList;
+    
+    var options = {
+        'data-http-request': getList,
+        'data-http-response': getList
+    };
+		
+    return function(request) {
+		return messageProcessor.getTypeStucture(request, options); 
+    }
+})();
+
+/**
+ * React class to display console messages
+ */
+var ServiceMessages = React.createClass({
+    _changeSelected: function(index) {
+        
+    },
+    render: function() {
+        var dataHttpRequestMessages = this.props.dataHttpRequestMessages;
+        var dataHttpResponseMessages = this.props.dataHttpResponseMessages;
+        
+        // get child items
+        var requestItems = [];
+        for (var i = 0; i < dataHttpRequestMessages.length; i++) {
+            // TODO: replace with correlation matching once inplace
+            var httpRequest = dataHttpRequestMessages[i].payload;
+            var httpResponse = dataHttpResponseMessages[i].payload;
+            
+            requestItems.push(
+                <tr onClick={this.props.onSelectedIndex.bind(this, i)} className={classNames({
+                        'selected': i == this.props.selectedIndex
+                    })}> 
+                    <td>{httpRequest.url}</td>
+                    <td>{httpResponse.statusCode}</td>
```

Comment:
Good question. At the moment I'm showing what ever has come from the server. What formatting we want to put around that should be brought up. Given timing that is probably out of the scope of this PR.